hooks.go

```diff
@@ -22,9 +22,14 @@ func (c LogConfig) initLogstashHook(h LogHook) error {
 		return ErrFailedToConfigureLogHook
 	}
 
-	formatter := getLogstashFormatter(h.Settings)
+	formatter := getLogstashFormatter(h.Settings).(*logrustash.LogstashFormatter)
```

Comment:
shouldn't we catch cast error here?

Reply:
no - check the implementation, and I have a test for this =)